tests/test-input-metadata/conda-lock.lock

```diff
@@ -0,0 +1,192 @@
+# This lock file was generated by conda-lock (https://github.com/conda-incubator/conda-lock). DO NOT EDIT!
+#
+# A "lock file" contains a concrete list of package versions (with checksums) to be installed. Unlike
+# e.g. `conda env create`, the resulting environment will not change as new package versions become
+# available, unless you explicitly update the lock file.
+#
+# Install this environment as "YOURENV" with:
+#     conda-lock install -n YOURENV --file conda-lock.lock
+# To update a single package to the latest version compatible with the version constraints in the source:
+#     conda-lock lock --lockfile conda-lock.lock --update PACKAGE
+# To re-solve the entire environment, e.g. after changing a version constraint in the source file:
+#     conda-lock -f /Users/c13371nroach/Documents/repos/conda-lock/tests/zlib/environment.yml --lockfile conda-lock.lock
+metadata:
+  channels:
+  - url: conda-forge
+    used_env_vars: []
+  - url: defaults
+    used_env_vars: []
+  content_hash:
+    linux-64: 1173e3c96ce20d063a5701b325b76deb97394f891af270af4ee0cb7cc1f6e838
+    osx-64: d01c1f5433f30bdbcd3bdad8d9b096774ab55f1210c094acdc61a35b32b28d67
+    win-64: 310b23581083bfb983927c40d3bdc86162192d7b26ffd7bffc385f627c155697
+  inputs_metadata:
+    /Users/c13371nroach/Documents/repos/conda-lock/tests/zlib/environment.yml:
```

Comment:
This should probably be a relative path not an absolute one

Reply:
I think (and I will x2 check this) That it uses a relative path if provided a relative path when conda-lock is invoked, and an absolute one if passed an absolute path. Happy to make the path relative (perhaps to the lockfile?) if you think that's always preferable.

Reply:
I think that is generally preferable.  Leaking in usernames etc into the lockfiles seems undesirable